eng/testing/WasmRunnerTemplate.sh

```diff
@@ -10,7 +10,8 @@ XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 dotnet xharness wasm test --engine=$JAVASCRIPT_ENGINE \
     --js-file=runtime.js -v \
     --output-directory=$XHARNESS_OUT \
-    -- --enable-gc --run WasmTestRunner.dll ${@:2}
+    -- --enable-gc --enable-zoneinfo \
```

Comment:
Should `--enable-zoneinfo` be enabled for all projects?

The way we construct this file is changing in: https://github.com/dotnet/runtime/pull/37752/files#diff-cb1e9a952e91b5f53a90e69eb6536127R14

I think this should be more a settting in the project itself, and then we can include this argument in the template, just as we pass down the XunitArguments. Basically once https://github.com/dotnet/runtime/pull/37752 is merged, we could add something like:

```xml
 <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
    <!-- We need to set this in order to get extensibility on xunit category traits and other arguments we pass down to xunit via MSBuild properties -->
    <BrowserRuntimeArguments>--enable-gc</BrowserRuntimeArguments>
    <BrowserRuntimeArguments Condition="'$(BrowserEnableZoneInfo)' == 'true'">$(BrowserRuntimeArguments) --enable-zoneinfo</BrowserRuntimeArguments>
    <RunScriptCommand>$HARNESS_RUNNER xharness wasm test --engine=$(JSEngine) --js-file=runtime.js -v --output-directory=$XHARNESS_OUT -- $(BrowserRuntimeArguments) --run WasmTestRunner.dll $(AssemblyName).dll</RunScriptCommand>
  </PropertyGroup>
```

Somewhere around here: https://github.com/dotnet/runtime/pull/37752/files#diff-9e21cdbaea18a370997e662e572afc6dR16

Then the project could just set:
```xml
<BrowserEnableZoneInfo>true</BrowserEnableZoneInfo>
```

Reply:
I think merging this as is would break all test projects that don't have a reference to `System.Runtime.TestData` because it won't find the zoneinfo file when trying to read it:

```
read ("supportFiles/zoneinfo/mono-webassembly-zoneinfo-fs-smd.js.metadata", 'utf-8')
```

Reply:
That makes sense. I will wait for that PR to be merged and then will implement those changes.